lte/gateway/c/core/oai/common/CMakeLists.txt

```diff
@@ -63,26 +62,11 @@ add_subdirectory(glogwrapper)
 add_subdirectory(redis_utils)
 
 add_library(COMMON ${COMMON_SRC})
-target_link_libraries(COMMON
-    lfds710
-    LIB_3GPP LIB_BSTR LIB_HASHTABLE LIB_ITTI glogwrapper
-    )
 
-## Find + link sentry if it exists
-## TODO we can enable by default after we migrate to ubuntu 20.04
-find_library(SENTRY_LIB NAMES sentry sentry-native)
-if (SENTRY_LIB)
-  # Include dir
-  find_path(SENTRY_INCLUDE_DIR
-      NAMES
-      sentry.h
-      )
-  target_link_libraries(COMMON ${SENTRY_LIB} MAGMA_CONFIG)
-  target_include_directories(COMMON PUBLIC
-      ${CMAKE_CURRENT_SOURCE_DIR}
-      $<TARGET_FILE_DIR:COMMON>
-      )
-endif ()
+target_link_libraries(COMMON
```

Comment:
@amarpad Did you use a tool to format your CMake changes way back?

Reply:
OK amar is out this week it seems, but the spacing looks a bit of here? (I can't tell if it's just the GitHub UI though. As long as it's consistent with the rest of the file I'm OK)

Reply:
It is consistent. There are no spacing changes here in fact, just append to a line.